### PR TITLE
Provide priority to block content

### PR DESCRIPTION
**Pull request by VladimirZaets**
## Issue

Fix https://github.com/hlxsites/merative/issues/230

## Test URLs
  
- Before: https://main--merative2--hlxsites.hlx.page/drafts/vzaets/biorasi-case-study
- After: https://related-resources-inplace-declaration--merative2--hlxsites.hlx.page/drafts/vzaets/biorasi-case-study

### CODE REVIEW

File: blocks/related-resources/related-resources.js [hunk 1]
```diff
@@ -33,10 +32,10 @@ async function setRowDetails(row, block) {
           break;
         case 'H4':
         case 'H5':
-          row.assetType = el.innerHTML;
+          row.assettype = el.innerHTML;
           break;
         case 'P':
-          row.description = `<p>${el.innerHTML}</p>${row.description}`;
+          row.description = `<p>${el.innerHTML}</p>`;
```
Comment (amol-anand) [inline blocks/related-resources/related-resources.js:38]: what if there are multiple paragraphs of text? then you're only picking the last <p> tag essentially. You might want to concatenate the <p> tags into the description.

File: blocks/teaser-list/teaser-list.js [hunk 2]
```diff
@@ -67,8 +67,7 @@ export default async function decorate(block) {
   const pageList = await lookupPages(pathnames);
   if (pageList.length) {
     pageList.forEach((row) => {
-      // If the URL was not in the index, it is curated. Let's get the content differently
-      if (row.title === undefined) setRowDetails(row, blockCopy);
+      setRowDetails(row, blockCopy);
```
Comment (arumsey) [inline blocks/teaser-list/teaser-list.js:70]: Does this allow for any of eyebrow, title, description or link title to be overriden? I don't think eybrow can be overridden and linktext should only be overriden if its inner text !== its href.

File: blocks/related-resources/related-resources.js (hunk 1 above)
Comment (venturewise) [reply blocks/related-resources/related-resources.js:38]: Here's an example of multiple paragraphs in a card, FYI https://www.merative.com/support/imaging-product-support-details/tuesdays-with-merative-imaging

File: blocks/teaser-list/teaser-list.js (hunk 2 above)
Comment (venturewise) [reply blocks/teaser-list/teaser-list.js:70]: Elaborating on Anthony's comment:
- Teaser-list doesn't use eyebrows, so we're OK there ✔️ 
- When only headline + link is present, then description will auto populate ✔️ 
- When only description + link is present, then headline will auto populate ✔️ 
- When more than 1 paragraph is authored, only the 1st one shows 🆗 (I don't think teaser-list is expected to show more than 1).
- when inner text == its href, then "Learn more about" should be the default link text ❌

**Review by venturewise — CHANGES_REQUESTED** [review]

File: blocks/related-resources/related-resources.js [hunk 3]
```diff
@@ -33,15 +34,19 @@ async function setRowDetails(row, block) {
           break;
         case 'H4':
         case 'H5':
-          row.assetType = el.innerHTML;
+          row.assettype = el.innerHTML;
           break;
         case 'P':
-          row.description = `<p>${el.innerHTML}</p>${row.description}`;
+          description = `<p>${el.innerHTML}</p> ${description}`;
```
Comment (venturewise) [inline blocks/related-resources/related-resources.js:40]: There's an extra paragraph tag showing up on the 2nd card of each row I think.
image

**Review by arumsey — APPROVED** [review]
@venturewise Do you see any other issues? I am unable to view the source document.

File: blocks/related-resources/related-resources.js (hunk 3 above)
Comment (venturewise) [reply blocks/related-resources/related-resources.js:40]: Update: actually this is not a new regression, it's there on the Main branch as well. https://main--merative2--hlxsites.hlx.page/drafts/chelms/4-ways-case-study

Comment (venturewise) [reply blocks/related-resources/related-resources.js:40]: I will resolve this convo and you can make a new ticket for that.

**Review by venturewise — APPROVED** [review]

**Pull request merged**